app-backend/api/src/main/resources/application.conf

```diff
@@ -31,6 +31,8 @@ auth0 {
 
   groundworkConnectionName = "groundwork-db"
   groundworkConnectionName = ${?AUTH0_GROUNDWORK_CONNECTION_NAME}
+
+  erConnectionName = ${?AUTH0_ER_CONNECTION_NAME}
```

Comment:
It might be better to name this for its purpose, e.g., `anonymizedUserCreateConnectionName`, so that in the future we don't have to guess at whether the abbreviation is significant